.pre-commit-config.yaml

```diff
@@ -44,8 +44,8 @@ repos:
     rev: stable
     hooks:
     -   id: black
-        language_version: python3.6
-        args: ['--target-version', 'py36']
+        language_version: python3.7
+        args: ['--target-version', 'py37']
```

Comment:
we can probably skip this as it's in `pyproject.toml`?

Reply:
Let's do that in another PR, we already have had too much pain with `black` and Python version which it uses.